src/suggestionlistworker.cpp

```diff
@@ -47,7 +70,7 @@ void SuggestionListWorker::run()
             suggestionList.append(m_text + " (" + gt("fulltext-search") + ")");
             urlList.append(url);
         }
-    } else {
+    } catch (std::out_of_range& e) {
```

Comment:
The `else` was here for the second `if (reader)`, and the second if was about fulltext search.
Now, the `catch` is about the only one `try` so the following comment is a bit odd.
It would be better to have:
```
// Impossible to find the requested archive (bug ?)
// We could propose a suggestion to do multi-zim search with:
// url.setHost("library.search");
// but we don't have a correct UI to select on which zim search, how to display results, ...
// So do nothing for now